extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaIndexTaskClient.java

```diff
@@ -622,7 +665,8 @@ private FullResponseHolder submitRequest(
             Thread.sleep(sleepTime);
           }
           catch (InterruptedException e2) {
-            Throwables.propagate(e2);
+            Thread.currentThread().interrupt();
+            throw new RuntimeException(e2);
```

Comment:
suggest `e.addSuppressed(e2)` and throwing the original exception

Reply:
Changed to throw original exception with addSuppressed